docs/src/pages/api/DayPicker.js

```diff
@@ -61,6 +61,7 @@ export default () => (
     <p>
       <a href="#renderDay">renderDay</a>, <a href="#renderWeek">renderWeek</a>,{' '}
       <a href="#weekdayElement">weekdayElement</a>,{' '}
+      <a href="#weekNumberElement">weekNumberElement</a>,{' '}
```

Comment:
I've added doc updates but the PR template instructs not to. Thought it might be useful but happy to revert this if needed